src/content/docs/accounts-partnerships/accounts/saml-single-sign/configure-saml-multiple-accounts.mdx

```diff
@@ -6,7 +6,10 @@ topics:
   - Accounts
   - Original accounts and billing
   - SAML SSO (original users)
-japaneseVersion: ''
+redirect_from:
+  - /docs/subscriptions/configuring-saml-with-multiple-accounts/
```

Comment:
Can you double check that trailing slashes redirect properly? I think we are using stripped paths when creating pages, so I'm not sure if this causes issues or not. It might be best to strip the trailing slash anyways since we want to encourage content authors to use the stripped slash.

Reply:
yep, will do. that was a requirement of that other plugin.